healthcare/api-client/v1/dicom/README.rst

```diff
@@ -122,15 +122,14 @@ To run this sample:
                             'roles/editor'
         export-dicom-store  Export data to a Google Cloud Storage bucket by
                             copying it from the DICOM store.
-        import-dicom-store  Import data into the DICOM store by copying it from
+        import-dicom-store  Imports data into the DICOM store by copying it from
                             the specified source.
 
     optional arguments:
       -h, --help            show this help message and exit
       --project_id PROJECT_ID
                             GCP project name
-      --cloud_region CLOUD_REGION
-                            GCP region
+      --location LOCATION   GCP region
```

Comment:
"GCP location" for consistence elsewhere.

Reply:
Done.